StyleBuilder.js

```diff
@@ -130,7 +131,13 @@ class StyleBuilder {
       [`border${side}Color`]: "initial",
     };
 
-    const values = value.split(" ");
+    const splitValues = value.split(" ");
+    const color = splitValues[2];
+
+    const values = color && _colorValuesThatCouldHaveSpaces.includes(color.split("(")[0])
+      ? [...splitValues.slice(0, 2), splitValues.slice(2, Infinity).join("")]
+      : splitValues
```

Comment:
I think you're right, and was the answer I was looking for. Anything between parens is really jusst part of the same value and can be stitched together to make the rest of the function work. I'm gonna sleep on it and update my PR. Thanks 😄